agents/gotToGoodBraitenberg.py

```diff
@@ -0,0 +1,71 @@
+from animalai.envs.actions import AAIActions, AAIAction
+from animalai.envs.raycastparser import RayCastParser
+from animalai.envs.raycastparser import RayCastObjects
+
+class Braitenberg():
+    """Implements a simple Braitenberg vehicle agent that heads towards food
+    Can change the number of rays but only responds to GOODGOALs, GOODGOALMULTI and BADGOAL"""
+    def __init__(self, no_rays):
+        self.no_rays = no_rays
+        assert(self.no_rays % 2 == 1), "Only supports odd number of rays (but environment should only allow odd number"
+        self.listOfObjects = [RayCastObjects.GOODGOAL, RayCastObjects.GOODGOALMULTI, RayCastObjects.BADGOAL, RayCastObjects.IMMOVABLE, RayCastObjects.MOVABLE]
+        self.raycast_parser = RayCastParser(self.listOfObjects, self.no_rays)
+        self.actions = AAIActions()
+        self.prev_action = self.actions.NOOP
+
+    def prettyPrint(self, obs) -> str:
+        """Prints the parsed observation"""
+        return self.raycast_parser.prettyPrint(obs)
+    
+    def get_action(self, obs) -> AAIAction:
+        """Returns the action to take given the current parsed raycast observation"""
+        obs = self.raycast_parser.parse(obs)
+        newAction = self.actions.NOOP
+        if self.ahead(obs, RayCastObjects.GOODGOALMULTI):
```

Comment:
Just to double check (because it does make sense): Assuming that multiple of these conditions can be true at the same time, forward was precedence of left and right?

Reply:
Yeah, forwards has precedence over left and right, thanks.